src/main/java/net/snowflake/client/jdbc/SnowflakeUtil.java

```diff
@@ -493,6 +493,11 @@ public static String systemGetProperty(String property) {
     }
   }
 
+  // This function is for mocking purpose. Because System static functions can't be mocked by mockito
+  public static String systemGetEnv(String env) {
```

Comment:
This is good wrapper. can you replace all of `System.getenv` with this?
See `systemGetProperty` about catching the SecurityException.

Reply:
I recall some strict environment doesn't allow to call any `System.getProperty` and potentially `System.getenv` as well.

Reply:
Np! Do you want me to do it in this pr? Or open a new pr for it?

Reply:
Let's do it in the PR.